### PR TITLE
IRSA-793-IRSA-798:add image type option to main panel  add image source panel option

### PR DESCRIPTION
**Pull request by lznakano**
IRSA-798
   Add image type options
   Add image source panel options
   Add 3 color panels
   Add Fits upload panel
   Add URL upload panel
   Enable FITS, URL and 3-color image search and display

Known issues:
  1. Need nested radio button groups, current implementation uses the first selected image sources in the array
 2. Need a None or Disable radio button to exclude a color panel.  Current implementation uses collapsible's open or close to exclude or include a color. 

To test it:
http://localhost:8080/firefly/firefly-dev.html
and then select Image Concept.

### CODE REVIEW

File: src/firefly/js/ui/ImageSelect.jsx [hunk 1]
```diff
@@ -290,34 +296,59 @@ const hasImageSelection = (groupKey, proj) => {
 
 
 // eslint-disable-next-line
-function BandSelect({subProject, projectData, labelMaxWidth}) {
+function BandSelect({subProject, multiSelect, projectData, labelMaxWidth}) {
     subProject = isNil(subProject) ? '' : subProject;
-    const fieldKey= `IMAGES_${get(projectData, [0, 'project'])}||${subProject}`;
+    const fieldKey= `IMAGES_${get(projectData, [0, 'project'])}` || `${subProject}`;
```
[the remark this answers is not in the document]
Comment (lznakano) [reply src/firefly/js/ui/ImageSelect.jsx]: I saw the key in many cases ended with || (for example SDSS||).  Do you intend the key ended with two bars?  I thought it was a mistake.  So I changed it.

**Comment by lznakano** [comment]
IRSA-988 was created.  It should fix the issues 1-3 Loi pointed out.

**Comment by ejoliet** [comment]
Actually, about the color tabs, they are redundant as you already have the tabs on top horizontally. Just remove them. The tab on top with 'red' blue, green are enough.

**Comment by lznakano** [comment]
I fixed the image source and type option panel's layout.  I pushed in,  in order for you to take a look and let me know if it looks OK.   I am still working on the file upload issue Loi pointed out.

**Comment by ejoliet** [comment]
Closing this PR, branch pushed was corrupted.

**Pull request closed without merging**